bool use_default_target
bool use_named_target
string planning_group
string named_target
string robot
geometry_msgs/PoseStamped target_pose

---

int64 code
string message
temoto_core/Error[] error_stack
